pysteps/utils/reprojection.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from rasterio import Affine as A
+from rasterio.warp import reproject, Resampling
+import xarray as xr
+import numpy as np
+
+
+def reprojection(R_src, R_dst):
+    """Reprojects precipitation fields to the domain of another precipiation
+    field.
+
+    Parameters
+    ----------
+    R_src: xarray
```

Comment:
here and elsewhere, specify whether you expect an xarray.Dataset or xarray.DataArray

```suggestion
    R_src: xr.DataArray
```